hwtest/pgraph_scan.cc

```diff
@@ -107,7 +107,14 @@ class ScanDebugTest : public ScanTest {
 			bitscan(0x4000a0, 0xffffffff, 0);
 			bitscan(0x4000a4, 0xf, 0);
 			bitscan(0x4000c0, 0x1e, 0);
+		} else {
+			bitscan(0x400080, 0x00000107, 0);
+			bitscan(0x400084, 0x7010c7c1, 0);
+			bitscan(0x40008c, 0xe1fad155, 0);
+			bitscan(0x400090, 0x33ffffff, 0);
+			bitscan(0x4000c0, 0x00000006, 0);
```

Comment:
Please write which NV4x you tested this on - I'm going to guess it's wildly different between chipsets.